### PR TITLE
Issue #82: Fix how peers and events are seeing

### PR DESCRIPTION
**Pull request by AgustinCB**

### CODE REVIEW

File: src/proxy/internal/Makefile [hunk 1]
```diff
@@ -6,3 +6,6 @@ clean:
 
 proto:
 	protoc --go_out=plugins=grpc,import_path=internal:. *.proto
+
+build:
+	exit 0
```
Comment (SamuelMarks) [inline src/proxy/internal/Makefile]: So basically a `nop`?

Comment (AgustinCB) [reply src/proxy/internal/Makefile]: Yep, without this, `make build` complains that there's no `build` target. `Make` can be pretty awful :D

**Comment by AgustinCB** [comment]
Uhm, this is missing this commit: https://github.com/AgustinCB/lachesis/commit/b219ff904fb883b76ed392d6e7914e7bc77644a3. Not sure why it's not appearing in this pr, as it's in the same branch...

**Comment by SamuelMarks** [comment]
🤔, try sending a new PR

**Pull request merged**